components/request_otr/browser/BUILD.gn

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//brave/build/config.gni")
+import("//brave/components/request_otr/common/buildflags/buildflags.gni")
+
+source_set("browser") {
+  sources = []
+  deps = []
+
+  if (enable_request_otr) {
+    sources += [
+      "request_otr_component_installer.cc",
+      "request_otr_component_installer.h",
+      "request_otr_controller_client.cc",
+      "request_otr_controller_client.h",
+      "request_otr_navigation_throttle.cc",
+      "request_otr_navigation_throttle.h",
+      "request_otr_page.cc",
+      "request_otr_page.h",
+      "request_otr_rule.cc",
+      "request_otr_rule.h",
+      "request_otr_service.cc",
+      "request_otr_service.h",
+      "request_otr_tab_storage.cc",
+      "request_otr_tab_storage.h",
+    ]
+
+    deps += [
+      "//base",
+      "//brave/components/brave_component_updater/browser",
+      "//brave/components/brave_shields/browser:browser",
+      "//brave/components/l10n/common:common",
+      "//brave/components/request_otr/common:common",
+      "//brave/components/resources:static_resources_grit",
+      "//brave/components/resources:strings_grit",
+      "//brave/extensions:common",
+      "//components/prefs:prefs",
+      "//components/security_interstitials/content:security_interstitial_page",
+      "//components/security_interstitials/core:core",
+      "//components/user_prefs:user_prefs",
+      "//content/public/browser",
+      "//extensions/browser",
```

Comment:
`enable_extensions` guard

Reply:
based on discussions, removing this entirely instead